app/controllers/reservations_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ReservationsController < ApplicationController
+  before_action :authenticate_user!
+  before_action :query_resesrvation, except: :mine
```

Comment:
- Kindly add your `:create` action to this `:except` definition to prevent the before effect on the action.